src/SLCore/Service/Connection/ListUserOrganizationsResponse.cs

```diff
@@ -0,0 +1,27 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Diagnostics.CodeAnalysis;
+using SonarLint.VisualStudio.SLCore.Service.Connection.Models;
+
+namespace SonarLint.VisualStudio.SLCore.Service.Connection;
+
+[ExcludeFromCodeCoverage]
```

Comment:
We should have a test for this one as well. I believe it's kinda a convention, to check if it searilizes correctly.

Reply:
I think we should discuss this point. It feels pointless to test the JsonConvert from Newtonsoft over and over again, especially for records with simple structure.

Reply:
I added it. We might add a more generic test using reflection, which will make these kind of tests obsolete, but it should probably be a separate PR